statuspage/src/main/java/com/jagrosh/jdautilities/statuspage/endpoints/ScheduledMaintenances.java

```
@@ -0,0 +1,132 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.jagrosh.jdautilities.statuspage.endpoints;
+
+import com.jagrosh.jdautilities.statuspage.data.Page;
+import com.jagrosh.jdautilities.statuspage.data.ScheduledMaintenance;
+
+import javax.annotation.Nonnull;
+import javax.annotation.concurrent.Immutable;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.ListIterator;
+
+@Immutable
+public class ScheduledMaintenances implements List<ScheduledMaintenance>
+{
+    @Nonnull
+    protected final Page page;
+    @Nonnull
+    protected final List<ScheduledMaintenance> ScheduledMaintenances;
+
+    public ScheduledMaintenances(@Nonnull Page page, @Nonnull List<ScheduledMaintenance> ScheduledMaintenances)
+    {
+        this.page = page;
+        this.ScheduledMaintenances = ScheduledMaintenances;
+    }
+
+    @Nonnull
+    public Page getPage()
+    {
+        return page;
+    }
+
+    @Nonnull
+    public List<ScheduledMaintenance> getScheduledMaintenances()
+    {
+        return ScheduledMaintenances;
+    }
+
+    @Override
+    public int size() {return ScheduledMaintenances.size();}
+
+    @Override
+    public boolean isEmpty() {return ScheduledMaintenances.isEmpty();}
+
+    @Override
+    public boolean contains(Object o) {return ScheduledMaintenances.contains(o);}
+
+    @Override
+    @Nonnull
+    public Iterator<ScheduledMaintenance> iterator() {return ScheduledMaintenances.iterator();}
+
+    @Override
+    @Nonnull
+    public Object[] toArray() {return ScheduledMaintenances.toArray();}
+
+    @Override
+    @Nonnull
+    @SuppressWarnings("SuspiciousToArrayCall")
+    public <T> T[] toArray(@Nonnull T[] a) {return ScheduledMaintenances.toArray(a);}
+
+    @Override
+    public boolean add(ScheduledMaintenance ScheduledMaintenance) {return ScheduledMaintenances.add(ScheduledMaintenance);}
```

This class is marked as `@Immutable` yet we are allowing modification of the underlying list?
This should throw `UnsupportedOperationException` or the `List` interface should not be implemented at all.
(This goes for all other methods that might modify this object)